main.js

```diff
@@ -1,3 +1,3 @@
 console.log("Hello from sub_brance_3!");
-console.log("Edit 1 from main...RAHHHHHHH")
+console.log("Edit 1 from main...dih")
```

Comment:
my dih is too small, make it BIG
```suggestion
console.log("Edit 1 from main...DIH")
```